addons/binding/org.openhab.binding.pioneeravr/ESH-INF/i18n/pioneeravr_xx_XX.properties

```diff
@@ -0,0 +1,11 @@
+# binding

```

Comment:
please remove this file, it is only a template if you want to provide other locales.